competition-simulator.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: page
+title: Competition Simulator
+---
+
+Competition Simulator
+============
+
+For SR2020, a virtual competition will be run using a brand new simulator.
+
+<div class="info">
+  This is not the same as the <a href="/docs/programming/simulator/">regular simulator</a>.
+</div>
+
+## Installation
+
+### Prerequisites
+
+This simulator is based off [Webots](https://cyberbotics.com/#download), which will need to be downloaded and installed.
+
+You will also need Python installed. The simulator supports >=3.4.
```

Comment:
This conflicts with the dev instructions in https://github.com/srobo/competition-simulator/pull/31, which indicates that this is Python 2.7.

Reply:
Fixed the linked guide accordingly. We've opted to go with Python 3 because Python 2 seems quite buggy in WeBots